public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -108,6 +109,12 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
       notifyApp.error(conditionErrorMsg);
       return;
     }
+    // if this rule is using manual routing, we save this option in local storage
```

Comment:
I decided to put the logic in here, as it's enough to know that user had the intention to save the rule using simplified routing.